site/_posts/2020-09-10-building-profile-portal-with-flask-oauth-apis.md

```diff
@@ -127,7 +154,9 @@ if __name__ == "__main__":
   app.run(debug=True)
 ```
 
-You are going to use flask templates to create the HTML pages. Create a directory called `templates` and a file called `index.html` in that directory. Put the following markup in there:
+At this point we have not created the `settings.txt` file so you will need to comment out the line `app.config.from_object('settings.Config')`.
```

Comment:
If including this line is busted, let's just remove it at this point in the tutorial.